src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -96,6 +96,16 @@ public class Win32PlatformOptions
         /// This can be useful when you need a rounded-corner blurred Windows 10 app, or borderless Windows 11 app
         /// </summary>
         public float? CompositionBackdropCornerRadius { get; set; }
+
+        /// <summary>
+        /// When <see cref="UseDxgiSwapChain"/> is active, renders Avalonia through a low-latency Dxgi Swapchain.
+        /// Requires Feature Level 11_3 to be active, Windows 8.1+ Any Subversion. 
+        /// This is only recommended if low input latency is desirable, and there is no need for the transparency
+        /// and stylings / blurrings offered by <see cref="UseWindowsUIComposition"/><br/>
+        /// This is mutually exclusive with 
+        /// <see cref="UseWindowsUIComposition"/> which if active will override this setting. 
+        /// </summary>
+        public bool UseDxgiSwapChain { get; set; } = false;
```

Comment:
I think it should be `UseLowLatencyDxgiSwapChain` since ANGLE already uses a regular one by default.

Reply:
Good suggestion, taken, renamed option.